xla/autotuning.proto

```diff
@@ -98,7 +115,9 @@ message AutotuneResult {
     stream_executor.dnn.AlgorithmProto algorithm = 16;
   }
 
-  // Next ID: 17
+  string fusion_name = 18;
```

Comment:
Storing fusion_name is probably a separate change?

Reply:
do you mean a separate PR or commit?

Reply:
PR.